go-tipb/expression.pb.go

```diff
@@ -33,6 +33,10 @@ var _ = proto.Marshal
 var _ = fmt.Errorf
 var _ = math.Inf
 
+// This is a compile-time assertion to ensure that this generated file
+// is compatible with the proto package it is being compiled against.
+const _ = proto.ProtoPackageIsVersion1
```

Comment:
Seems this line is deleted in this PR: https://github.com/pingcap/tipb/pull/6/files
Maybe we use different protoc.